src/javascript/app/common/content_visibility.js

```diff
@@ -47,16 +46,26 @@ const mt_company_rule   = 'mtcompany';
 
 const ContentVisibility = (() => {
     const init = () => {
-        if (Client.isLoggedIn()) {
+        const client_is_logged_in = Client.isLoggedIn();
+        if (client_is_logged_in) {
             BinarySocket.wait('authorize', 'landing_company').then(() => {
+                const is_virtual =  State.getResponse('authorize.is_virtual');
+                const landing_company = is_virtual ?
```

Comment:
shouldn't it be `!is_virtual`?

Reply:
if the user is virtual we want to know what's the landing company for real accounts, because in authorize response landing company is `virtual` so we can get it from landing company call, and if user is not virtual we get landing company for the account from authorize response